.github/workflows/ci.yml

```
@@ -295,24 +295,24 @@ jobs:
-          sync-labels: true
-      
-      - name: Auto-assign Reviewers
-        uses: kentaro-m/auto-assign-action@v1.2.5
-        with:
-          repo-token: ${{ secrets.GITHUB_TOKEN }}
-          configuration-path: .github/auto-assign.yml
+  # pr-automation:
+  #   name: PR Automation
+  #   runs-on: ubuntu-latest
+  #   if: github.event_name == 'pull_request'
+  #   
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)